src/components/PersonName/PersonName.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useState } from 'react';
+import { NavLink, useRouteMatch, useLocation } from 'react-router-dom';
+
+import { Identity } from '../../utils/type';
+import './PersonName.scss';
+
+export const PersonName: React.FC<Identity> = ({ human, slug, people }) => {
+  const match = useRouteMatch();
+  const { search } = useLocation();
+  const [subject, setSubject] = useState<any>('');
+
+  useEffect(() => {
+    setSubject(people.find(individ => individ.name === human) || undefined);
```

Comment:
```suggestion
    setSubject(people.find(individ => individ.name === human));
```
find вернет `undefined`, если ничего не найдет